.github/workflows/validate.yml

```diff
@@ -125,3 +125,23 @@ jobs:
       run: |
         . /etc/profile.d/lmod.sh
         tests/ci/run_build.py ohpc ${{ steps.files.outputs.added_modified }}
+
+  build_on_centos_stream_9:
+    runs-on: ubuntu-latest
+    name: Build on CentOS Stream 9
+    container:
+      image: quay.io/centos/centos:stream9
+    steps:
+    - name: Install git
+      run: dnf -y install git
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Does it really need all history and all tags to be checked out ?
https://github.com/actions/checkout/blob/main/action.yml#L57

Reply:
not sure, probably just some historic copy and paste. also removed